src/index.ts

```diff
@@ -650,6 +651,7 @@ export async function precomputedInit(
     precomputedFlagStore: memoryOnlyPrecomputedFlagsStore,
     requestParameters,
     subject,
+    precomputedBanditStore: memoryOnlyPrecomputedBanditsStore,
```

Comment:
We already provide this in `offlinePrecomputedInit`, but it was missing here

https://github.com/Eppo-exp/js-client-sdk/blob/90f65159da8e75af66de5fb7ccb1a9e55e07c595/src/index.ts#L741-L745